### PR TITLE
dataset viewer on private datasets

### PR DESCRIPTION
**Pull request by severo**

### CODE REVIEW

File: docs/hub/datasets-viewer-configure.md [hunk 1]
```diff
@@ -25,3 +25,7 @@ viewer: false
 ```
 
 Note that the viewer is always disabled on the private datasets.
+
+## Private datasets
+
+The Dataset Viewer on private datasets is a feature available to [PRO users](https://huggingface.co/pricing) and [Enterprise organizations](https://huggingface.co/enterprise). With the Dataset Viewer, you can check the correctness of your private dataset before publishing it. You will also be able to share with your team benefits like searching, filtering, and conversion to Parquet for datasets that will remain private.
```
Comment (lhoestq) [inline docs/hub/datasets-viewer-configure.md:31]: remove first line + remove last line which feels redundant

```suggestion
## Private datasets

The Dataset Viewer on private datasets is a feature available to [PRO users](https://huggingface.co/pricing) and [Enterprise organizations](https://huggingface.co/enterprise).
```

File: docs/hub/enterprise-datasets.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Datasets
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+The Dataset Viewer is enabled on private datasets owned by an Enterprise organization. With the Dataset Viewer, you can check the correctness of your private dataset before publishing it. You will also be able to share with your team benefits like searching, filtering, and conversion to Parquet for datasets that will remain private.
```
Comment (lhoestq) [inline docs/hub/enterprise-datasets.md:7]: make it more entreprisy

```suggestion
The Dataset Viewer is enabled on private datasets owned by an Enterprise organization. The Dataset Viewer allows teams to understand their data and to help them build better data processing and filtering for AI. The Viewer allows to explore the datasets content, inspect data distributions, filter by values and even search for keywords. It also includes the datasets conversion to Parquet which can be used for programmatic data visualization.
```

**Review by lhoestq — APPROVED** [review]
LGTM ! just 2 comments:

File: docs/hub/enterprise-hub.md [hunk 3]
```diff
@@ -9,3 +9,4 @@ In this section we will document the following Enterprise Hub features:
 - [SSO](./enterprise-sso)
 - [Audit Logs](./audit-logs)
 - [Storage Regions](./storage-regions)
+- [Datasets](./enterprise-datasets)
```
Comment (julien-c) [inline docs/hub/enterprise-hub.md:12]:
```suggestion
- [Dataset viewer for Private datasets](./enterprise-datasets)
```

File: docs/hub/enterprise-datasets.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Datasets
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+The Dataset Viewer is enabled on private datasets owned by an Enterprise organization. The Dataset Viewer allows teams to understand their data and to help them build better data processing and filtering for AI. The Viewer allows to explore the datasets content, inspect data distributions, filter by values and even search for keywords. It also includes the datasets conversion to Parquet which can be used for programmatic data visualization.
```
Comment (julien-c) [inline docs/hub/enterprise-datasets.md:7]:
```suggestion
The Dataset Viewer is enabled on private datasets owned by an Enterprise Hub organization. 

The Dataset Viewer allows teams to understand their data and to help them build better data processing and filtering for AI. The Viewer allows to explore the datasets content, inspect data distributions, filter by values and even search for keywords. It also includes the datasets conversion to Parquet which can be used for programmatic data visualization.
```

(the product name is Enterprise Hub not Enterprise)

maybe add a visual screenshot too?

File: docs/hub/datasets-viewer.md [hunk 5]
```diff
@@ -53,4 +53,6 @@ For the biggest datasets, the page shows a preview of the first 100 rows instead
 To have a properly working Dataset Viewer for your dataset, make sure your dataset is in a supported format and structure.
 There is also an option to configure your dataset using YAML.
 
+For private datasets, the Dataset Viewer is a feature available to [PRO users](https://huggingface.co/pricing) and [Enterprise organizations](https://huggingface.co/enterprise).
```
Comment (julien-c) [inline docs/hub/datasets-viewer.md:56]:
```suggestion
For **private** datasets, the Dataset Viewer is enabled for [PRO users](https://huggingface.co/pricing) and [Enterprise Hub organizations](https://huggingface.co/enterprise).
```

File: docs/hub/datasets-viewer-configure.md [hunk 6]
```diff
@@ -23,5 +23,6 @@ The dataset viewer can be disabled. To do this, add a YAML section to the datase
 viewer: false
 ---
 ```
+## Private datasets
 
-Note that the viewer is always disabled on the private datasets.
+The Dataset Viewer on private datasets is a feature available to [PRO users](https://huggingface.co/pricing) and [Enterprise organizations](https://huggingface.co/enterprise).
```
Comment (julien-c) [inline docs/hub/datasets-viewer-configure.md:28]: same wording

File: docs/hub/datasets-adding.md [hunk 7]
```diff
@@ -96,7 +96,7 @@ Other formats and structures may not be recognized by the Hub.
 ### Dataset Viewer
 
 The [Dataset Viewer](./datasets-viewer) is useful to know how the data actually looks like before you download it.
-It is enabled by default for all public datasets.
+It is enabled by default for all public datasets. It is also available for private datasets owned by a [PRO user](https://huggingface.co/pricing) or an [Enterprise organization](https://huggingface.co/enterprise).
```
Comment (julien-c) [inline docs/hub/datasets-adding.md:99]: this one is good

File: docs/hub/_toctree.yml [hunk 8]
```diff
@@ -271,6 +271,8 @@
         title: Audit Logs
       - local: storage-regions
         title: Storage Regions
+      - local: enterprise-datasets
```
Comment (julien-c) [inline docs/hub/_toctree.yml:274]:
```suggestion
      - local: enterprise-hub-datasets
```
(maybe, up to you)

**Review by julien-c — APPROVED** [review]
lgtm with the suggested tweaks!

File: docs/hub/datasets-adding.md (hunk 7 above)
Comment (severo) [reply docs/hub/datasets-adding.md]: added "Hub"

File: docs/hub/enterprise-datasets.md (hunk 4 above)
Comment (severo) [reply docs/hub/enterprise-datasets.md]: done

**Comment by severo** [comment]
thanks for the reviews and improvements

**Pull request merged**